src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,34 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name) {
+    super(name);
+    this.hidden = false;
+  }
+  hide() {
+    this.hidden ? this.hidden = false : this.hidden = true;
```

Comment:
didn't see it. Use it

![image](https://user-images.githubusercontent.com/64437769/148079021-949283e8-2d8d-404c-b3f9-061cdccb051d.png)